New log file opened at Fri Jun 03 16:02:58 ICT 2016 by 7472@Rifqi
Taking next connection to: 57551
New client attempting to connect on port: 57552-57551
  Address = /192.168.136.1:57551
This client on port 57552 has uuid 85735705-daa6-4a80-9d7c-5e10c719f8f7 configName activator humanReadableName Activator protocolVersion 3 featureTags Vector()
Connected Clients: activator-85735705-daa6-4a80-9d7c-5e10c719f8f7
Taking next connection to: 57551
Checking to see if clients are empty...
We have 1 non-daemon clients, continuing serving connections.
Taking next connection to: 57551
Checking to see if clients are empty...
We have 1 non-daemon clients, continuing serving connections.
Taking next connection to: 57551
New client attempting to connect on port: 57605-57551
  Address = /192.168.136.1:57551
Handshake exception on socket: 57605
sbt.impl.ipc.HandshakeException: Unable to perform handshake
	at sbt.impl.ipc.Peer.handshake(IPC.scala:46)
	at sbt.impl.ipc.MultiClientServer.<init>(IPC.scala:150)
	at sbt.server.SbtServerSocketHandler$$anon$1.run(SbtServerSocketHandler.scala:43)
Caused by: java.net.SocketException: Software caused connection abort: recv failed
	at java.net.SocketInputStream.socketRead0(Native Method)
	at java.net.SocketInputStream.socketRead(Unknown Source)
	at java.net.SocketInputStream.read(Unknown Source)
	at java.net.SocketInputStream.read(Unknown Source)
	at java.io.BufferedInputStream.fill(Unknown Source)
	at java.io.BufferedInputStream.read(Unknown Source)
	at java.io.DataInputStream.readInt(Unknown Source)
	at sbt.impl.ipc.Peer.receive(IPC.scala:95)
	at sbt.impl.ipc.Peer.handshake(IPC.scala:49)
	... 2 more
Taking next connection to: 57551
New client attempting to connect on port: 57606-57551
  Address = /192.168.136.1:57551
This client on port 57606 has uuid e22cb71c-1510-4f8a-b4e7-71c8421882c4 configName play-fork-run humanReadableName Play Fork Run protocolVersion 1 featureTags Vector()
Connected Clients: activator-85735705-daa6-4a80-9d7c-5e10c719f8f7, play-fork-run-e22cb71c-1510-4f8a-b4e7-71c8421882c4
Taking next connection to: 57551
Checking to see if clients are empty...
We have 1 non-daemon clients, continuing serving connections.
Taking next connection to: 57551
